eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServerV2.java

```diff
@@ -55,20 +56,17 @@
     private VariantWithSamplesAndAnnotationsService service;
 
     @GetMapping(value = "/{variantCoreString}")
-    public Resource<QueryResponse> getCoreInfo(@PathVariable("variantCoreString") String variantCoreString,
-                                               @RequestParam(name = "species") String species,
-                                               @RequestParam(name = "assembly") String assembly,
-                                               @RequestParam(name = "annot-vep-version", required = false)
-                                                       String annotationVepVersion,
-                                               @RequestParam(name = "annot-vep-cache-version", required = false) String
-                                                       annotationVepCacheVersion,
-                                               HttpServletResponse response) throws IllegalArgumentException {
+    public Resource getCoreInfo(@PathVariable("variantCoreString") String variantCoreString,
+                                @RequestParam(name = "species") String species,
+                                @RequestParam(name = "assembly") String assembly,
+                                HttpServletResponse response) throws IllegalArgumentException{
         initializeQuery();
-        String errorMessage = checkParameters(variantCoreString, annotationVepVersion, annotationVepCacheVersion,
-                species, assembly);
-        if (errorMessage != null) {
+        try {
+            checkParameters(variantCoreString, null, null, species, assembly);
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage());
```

Comment:
Although logging is nice in production, it should be implemented with a logger class, instead of just printing into the standard output.

Reply:
Sorry, I did this for debugging and missed to remove it.
I have removed this :)